src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -39,11 +39,18 @@ enum PAL_GssFlags : uint32_t
     PAL_GSS_C_DELEG_POLICY_FLAG = 0x8000
 };
 
+/*
+Issue: #7342
+Disable padded warning which occurs in case of arm32 builds
```

Comment:
I would update the comment to say that it is "32-bit builds" not just arm32 builds.